src/Assembly.js

```diff
@@ -23,10 +23,15 @@ class Assembly extends React.Component {
 
     this.network = new Network(process.env.REACT_APP_URL_API)
 
+    Aviator.setRoutes({ "/bigscreen": () => this.current_page = BigScreen })
+    Aviator.dispatch()
```

Comment:
The display is now purely controlled by the data in `Assembly`.
This `Aviator` line keeps us compatible with the bigscreen display – it'll be the only working URL from now on.